cvat/apps/iam/permissions.py

```diff
@@ -789,6 +790,8 @@ def get_scopes(request, view, obj):
             ('annotations', 'DELETE'): 'delete:annotations',
             ('annotations', 'PUT'): 'update:annotations',
             ('data', 'GET'): 'view:data',
+            ('data_info','GET'): 'view:data',
```

Comment:
`view:data`, `update:data` to `view:metadata` and `update:metadata`
I will recommend to rename data_info to metadata

Reply:
Changed

Reply:
Cannot rename `data_info` to `metadata` due to this check
<https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/iam/permissions.py#L1041:L1042>

Reply:
@ActiveChooN , can we make the check more strict? For example, check view.basename?

Reply:
Basename would be task or jobs in that case. But I don't really know why is this check even needed, can't find any mention of `metadata` in any view. I suggest simply delete second part of the check.

Reply:
https://www.django-rest-framework.org/api-guide/metadata/